Modelica/Units.mo

```diff
@@ -1537,8 +1537,8 @@ Real direction[3](unit=\"1\") = to_unit1(v);   // Automatically vectorized call
 
     function from_minute "Convert from minute to second"
       extends Modelica.Units.Icons.Conversion;
-      input Modelica.Units.NonSI.Time_minute minute "minute value";
-      output SI.Time s "second value";
+      input Modelica.Units.NonSI.Time_minute minute "Value in minute";
+      output SI.Time s "Value in second";
```

Comment:
```suggestion
      input Modelica.Units.NonSI.Time_minute minute "Value in minute";
      output SI.Time s "Value in second";
```

Should it be "Value in seconds" and "Value in minutes"? If yes, when to apply the plural?

Reply:
> Should it be "Value in seconds" and "Value in minutes"? If yes, when to apply the plural?

I was asking this myself many times when applying all these changes. I thought that the plural form in general sounds more correct than the singular form, but I was reluctant to change it, as it causes a massive change... 

One way to overcome this issue were to write "Value in unit second" instead of "Value in second". @beutlich what do you think?

Reply:
I am in favour of keeping the singular form.

Reply:
Do you mean we keep "Value in second" and do not change it to "Value in unit second"?

Reply:
Exactly.